cmd/swagger/commands/expand.go

```diff
@@ -10,7 +10,7 @@ import (
 	"github.com/go-openapi/spec"
 	"github.com/go-openapi/swag"
 	flags "github.com/jessevdk/go-flags"
-	yaml "gopkg.in/yaml.v2"
+	yaml "gopkg.in/yaml.v3"
```

Comment:
yaml v3 has removed MapSlice and thus would require a number of other changes. 

```suggestion
	yaml "gopkg.in/yaml.v3"
```
```suggestion
	yaml "gopkg.in/yaml.v2"
```